prow/README.md

```diff
@@ -98,7 +98,7 @@ hard requirement.
  gcloud -q container node-pools create build-pool --project "${PROJECT}" --cluster "${CLUSTER}" --zone "${ZONE}" --machine-type n1-standard-8 --num-nodes 4 --local-ssd-count=1 --node-labels=role=build
  ```
 
-1. Create the secrets that allow prow to talk to GitHub. The `hmac-token` is
+2. Create the secrets that allow prow to talk to GitHub. The `hmac-token` is
```

Comment:
It's not rendering them right this way. https://github.com/kubernetes/test-infra/blob/master/prow/README.md